src/cloud/components/onboarding/EngagementLink.tsx

```diff
@@ -0,0 +1,47 @@
+// Libraries
+import {FC, useEffect} from 'react'
+import {useLocation} from 'react-router-dom'
+import {connect, ConnectedProps} from 'react-redux'
+import {get} from 'lodash'
+
+// Utils
+import {getOrg} from 'src/organizations/selectors'
+
+// Types
+import {AppState} from 'src/types'
+
+type ReduxProps = ConnectedProps<typeof connector>
+type Props = ReduxProps
+
+const EngagementLink: FC<Props> = ({org, me}) => {
+  const pathname = useLocation().pathname
+
+  useEffect(() => {
+    sendToUserPilot()
+  }, [pathname, org, me])
+
+  const sendToUserPilot = (): void => {
+    const userpilot = get(window, 'userpilot')
```

Comment:
this `get` doesn't do anything unless window doesn't exist, but more importantly, we dont need lodash for this type of thing anymore. typescript provides the ability to do this with the format: `const userpilot = window?.userpilot`. you can chain it on even longer requests like `const origin = window?.location?.origin || 'localhost'`. this will bail out anytime a property is missing and hit the `or` condition as a default value